huaweicloud/services/elb/resource_huaweicloud_elb_security_policy.go

```diff
@@ -0,0 +1,313 @@
+package elb
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+const (
+	stringRegxp     = `^[\x{4E00}-\x{9FFC}A-Za-z-_0-9?'#().,&%@!]*$`
+	stringFormatMsg = "Only letters, Chinese characters, digits, hyphens (-), underscores (_) and" +
+		" the following special characters are allowed: ?'#().,&%@!"
+)
+
+func ResourceSecurityPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceSecurityPoliciesV3Create,
+		UpdateContext: resourceSecurityPoliciesV3Update,
+		ReadContext:   resourceSecurityPoliciesV3Read,
+		DeleteContext: resourceSecurityPoliciesV3Delete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"protocols": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				Description: `Specifies the TSL protocol list which the security policy select.`,
+			},
+			"ciphers": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				Description: `Specifies the cipher suite list of the security policy.`,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.All(
+					validation.StringLenBetween(1, 64),
+					validation.StringMatch(regexp.MustCompile(stringRegxp), stringFormatMsg),
+				),
+				Description: `Specifies the ELB security policy name.`,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.All(
+					validation.StringLenBetween(0, 128),
+					validation.StringMatch(regexp.MustCompile(stringRegxp), stringFormatMsg),
+				),
+				Description: `Specifies the description of the ELB security policy`,
+			},
+			"enterprise_project_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The enterprise project ID to which the Enterprise router belongs.`,
```

Comment:
keep the uniform format, add `Specifies` to this attr, or remove `Specifies` of others.
Same to doc

Reply:
done